types/react-joyride/index.d.ts

```diff
@@ -9,15 +9,11 @@ import React = require("react");
 export default class Joyride extends React.Component<Props, State> {
     constructor(props: Props);
 
-    addTooltip(data: Step): void;
-    start(autorun?: boolean): void;
+    reset(restart: boolean): void;
```

Comment:
looking at the docs make it seem like it's not optional, but the code itself should handle undefined perfectly fine. Can probably just make it optional in the typing